tools/dev-doctor/rootcmd.go

```diff
@@ -73,6 +73,28 @@ func rootCmdRun(cmd *cobra.Command, args []string) {
 			versionRegexp: regexp.MustCompile(`clang version (\d+\.\d+\.\d+)`),
 			minVersion:    &semver.Version{Major: 10, Minor: 0, Patch: 0},
 		},
+		&binaryCheck{
+			name:          "docker-server",
+			command:       "docker",
+			ifNotFound:    checkWarning,
+			versionArgs:   []string{"version", "--format", "{{ .Server.Version }}"},
+			versionRegexp: regexp.MustCompile(`(\d+\.\d+\.\d+)`),
+		},
+		&binaryCheck{
+			name:          "docker-client",
+			command:       "docker",
+			ifNotFound:    checkWarning,
+			versionArgs:   []string{"version", "--format", "{{ .Client.Version }}"},
+			versionRegexp: regexp.MustCompile(`(\d+\.\d+\.\d+)`),
+		},
+		&binaryCheck{
+			name:          "docker-buildx",
+			command:       "docker",
+			ifNotFound:    checkWarning,
+			versionArgs:   []string{"buildx", "version"},
+			versionRegexp: regexp.MustCompile(`github.com/docker/buildx v(\d+\.\d+\.\d+)`),
```

Comment:
I wonder if just `.*buildx...` instead of the the full `github.com/docker/buildx...` would be more future proof here.

Reply:
I agree that it potentially would be, but let's fix that if/when folk start using forks of github.com/docker/buildx.